iis-workflows/pom.xml

```diff
@@ -686,47 +606,49 @@
 	                    </execution>
 	                </executions>
 	            </plugin>
-	            <!--  -->
+	            
 				<!-- end of plugins required by integration test priming -->
 				<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-surefire-plugin</artifactId>
-					<version>2.11</version>
+					<version>2.18.1</version>
 					<dependencies>
 						<dependency>
 							<groupId>org.apache.maven.surefire</groupId>
 							<artifactId>surefire-junit47</artifactId>
-							<version>2.12</version>
+							<version>2.18.1</version>
 						</dependency>
 					</dependencies>
 					<configuration>
-						<skip>true</skip>
+						<excludedGroups>eu.dnetlib.iis.IntegrationTest</excludedGroups>
 					</configuration>
 					<executions>
 						<execution>
-							<id>unit-test</id>
-							<phase>test</phase>
+							<id>default-test</id>
 							<goals>
 								<goal>test</goal>
 							</goals>
-							<configuration>
-								<skip>false</skip>
-								<excludedGroups>eu.dnetlib.iis.IntegrationTest</excludedGroups>
-							</configuration>
 						</execution>
+					</executions>
+				</plugin>
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-failsafe-plugin</artifactId>
+					<version>2.18.1</version>
+					<executions>
 						<execution>
-							<id>integration-test</id>
-							<phase>integration-test</phase>
+							<id>default-integration-test</id>
 							<goals>
-								<goal>test</goal>
+								<goal>integration-test</goal>
+								<goal>verify</goal>
 							</goals>
 							<configuration>
-								<skip>false</skip>
-								<forkMode>always</forkMode>
+								<redirectTestOutputToFile>true</redirectTestOutputToFile>
 								<argLine>-Xmx1024m</argLine>
 								<systemPropertiesVariables>
 									<hadoop.log.dir>/tmp</hadoop.log.dir>
 									<oozie.data.dir>${oozie.data.dir}</oozie.data.dir>
+									<connectionProperties>${connectionProperties}</connectionProperties>
```

Comment:
Information that if the connectionProperties is null then default connection properties (with location of it) will be used, would be useful.